models/asset.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import (
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type AssetInterface interface {
+	GetId() primitive.ObjectID
+	Description() string
+	GetAssetType() AssetInterface
```

Comment:
What's the point of this method? 
In the implementations (audience, chart, insight), the actual object is returned

Reply:
This was forgotten in there (and in the implementations). I was experimenting with different approaches to fetching the object. This one was not useful and was thus abandoned.